tests/conftest.py

```diff
@@ -37,6 +37,10 @@ def tol():
     """Numerical tolerance for equality tests."""
     return float(os.environ.get("TOL", TOL))
 
+@pytest.fixture(scope="session")
+def tf_tol():
+    """Numerical tolerance for equality tests."""
+    return float(os.environ.get("TF_TOL", TF_TOL))
```

Comment:
Did all the TF tests require a larger tolerance suddenly?

Reply:
Just the following one